restic/internal/backup_test.py

```diff
@@ -7,6 +7,17 @@
 
 class BackupTest(unittest.TestCase):
 
+    @mock.patch.object(backup.command_executor, 'execute')
+    def test_backup_single_path_group_by(self, mock_execute):
+        mock_execute.return_value = '{}'
+
+        restic.backup(paths=['/tmp/dummy-file.txt'], group_by='host,tags')
+
+        mock_execute.assert_called_with([
+            'restic', '--json', 'backup', '/tmp/dummy-file.txt', '--group-by',
+            'host,tags'
```

Comment:
Can we make this a list of strings rather than a comma-separated string? The CLI accepts a string, but Python can more clearly represent a list than the shell can.